specs/core/0_beacon-chain.md

```diff
@@ -1856,10 +1832,10 @@ This section is divided into Normative and Informative references.  Normative re
 ## Normative
 
 ## Informative
-<a id="ref-casper-ffg"></a> _**casper-ffg**_  
+<a id="ref-casper-ffg"></a> _**casper-ffg**_
```

Comment:
These spaces are for formatting.

Reply:
Having trailing spaces is not sustainable, partly because some editors (such as mine) as programmed to remove trailing spaces. Would it be sufficient to re-add the space and remove the following new line?

Reply:
I added a `</br>` tag. It is much easier to mimic and will not lead to space cleaning editor modifications.